servicebox-app/src/main/resources/META-INF/resources/js/primaza.js

```diff
@@ -19,4 +19,13 @@
             }
         };
     });
+
+    // Error handling after validation errors
+    htmx.on("htmx:responseError",function(evt) {
+        var element = document.getElementById("response");
+        if (element != null) {
+            var errorDom = new DOMParser().parseFromString(evt.detail.xhr.response, "text/html");
+            element.innerHTML = errorDom.body.innerText;
```

Comment:
Why do you set the htmx innerHTML here instead of returning it as we do with claim ?

Reply:
HTMX does nothing when the status code is other than 2xx, that's why it's needed this special handling.

Reply:
Let's keep this code right now. I have posted the question on HTMX discord. We will see what they will reply